content/en/docs/tasks/configure-pod-container/configure-service-account.md

```diff
@@ -245,32 +257,25 @@ imagePullSecrets:
 - name: myregistrykey
 ```
 
-Finally replace the serviceaccount with the new updated `sa.yaml` file
-
-```shell
-kubectl replace serviceaccount default -f ./sa.yaml
-```
-
-Now, any new pods created in the current namespace will have this added to their spec:
+Now you've made those changes, any new Pods created in the current namespace
+will have this added to their spec:
 
 ```yaml
 spec:
   imagePullSecrets:
   - name: myregistrykey
 ```
 
-<!--## Adding Secrets to a service account.
-
-TODO: Test and explain how to use additional non-K8s secrets with an existing service account.
--->
-
 ## Service Account Token Volume Projection
 
 {{< feature-state for_k8s_version="v1.12" state="beta" >}}
 
 {{< note >}}
-This ServiceAccountTokenVolumeProjection is __beta__ in 1.12 and
-enabled by passing all of the following flags to the API server:
+ServiceAccountToken volume projection is __beta__ since Kubernetes v1.12,
+controlled by the `TokenRequestProjection`
+[feature gate](/docs/reference/command-line-tools-reference/feature-gates/).
+
+To use it, you will need to pass all of the following flags to the API server:
 
```

Comment:
- clean up note shortcode: ServiceAccountToken volume projection is controlled by the `TokenRequestProjection` ...
- To use this feature, pass the following flags to the API server: